### PR TITLE
Added a guide on IP banning.

### PR DESCRIPTION
**Pull request by xavwashere**
^^

If there is an easy way to do this, I'll spontaneously combust.
Ignore the "fixing a typo" and "nvm" commits, I tried to be big brain and fix someone's typo only to realize thats probably how it's meant to be, I'm just stupid :)

### CODE REVIEW

File: phpmyadmin/banning.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+5. As shown by my amazing highlighting, focus on the row with that person's username on it.
+
+6. Use the scrollbar at the bottom of the screen to scroll to the other end of the page.
+
+7. You should see their IP somewhere on that row; copy it, we're going to need it in a moment. Also, try not to leak everyone's IP. It's not nice.
+
+![image](https://github.com/xavwashere/community-guide/assets/97399129/214c31f2-ee7b-4c02-b1dd-b8ea380f747c)
+
+8. You are now done in the `users` section. *loud confetti noises* Now, let's move on. Focus on the sidebar for a minute and find `bannedips`. Click on it.
```
Comment (Redactedslair) [inline phpmyadmin/banning.md]: i like the *loud confetti noises*

**Comment by Drtrayblox7855** [comment]
everything  seems good. just wait a day or 2 once Rya or MathieuAR will merge it.

File: phpmyadmin/banning.md (hunk 1 above)
Comment (xavwashere) [reply phpmyadmin/banning.md:31]: very fun

File: phpmyadmin/banning.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+9. You should see this screen; Click on the `Insert` button at the top of your screen (the button that is circled)
+
+![image](https://github.com/xavwashere/community-guide/assets/97399129/24048cee-04e2-4883-a3bf-d0cc0f70256e)
+
+10. There should be a table. Find the IP row and edit the value. It *should* already say `127.0.0.1`. Delete it and paste in the user's IP but **don't** press `ENTER` once you're done. Just click onto the text box underneath (ID).
+
+![image](https://github.com/xavwashere/community-guide/assets/97399129/4fcc74bf-e7c3-4065-8b03-4cf7866c0364)
+
+11. Next, you'll need to find the user's ID. You can find this by going to their profile in your GDPS and looking at the bottom of the window. You should be able to click on it and it will copy it.
```
Comment (Jecket22) [inline phpmyadmin/banning.md]: You sure this is in vanilla GD? I didn't find it on my end so I can't tell if you're using a mod of I am looking at the wrong place. Mind providing an alternative in this case?

**Comment by xavwashere** [comment]
why does it have the question label? @Jecket22

**Review by Jecket22 — APPROVED** [review]

**Pull request merged**